<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addLevel</class>
 <widget class="QDialog" name="addLevel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>570</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>570</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
font: 13pt &quot;微软雅黑&quot;;
background-color:rgb(255, 255, 255)
}
QLabel{
color:rgb(65, 65, 65);
font: 13pt &quot;微软雅黑&quot;;
}
QLineEdit {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
	font: 10pt &quot;微软雅黑&quot;
}
QComboBox {
    border-width: 1px;
    border-style: solid;
    border-radius: 4px;
	border-color:rgb(199, 199, 199);
	background-color:rgb(255, 255, 255);
	font: 13pt &quot;微软雅黑&quot;;
}
QComboBox::drop-down {
     subcontrol-origin:paddind;  
     subcontrol-position: top right;
     width: 15px;
     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px;
     border-bottom-right-radius: 3px;
	 border:none
 }

QComboBox::down-arrow {
     image: url(:/images/down.png);
     right: 3px;
}
QPushButton{font: bold 9pt &quot;微软雅黑&quot;;
color:rgb(140, 140, 140)}</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_67">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_98">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>赠送积分：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_85">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>享受折扣：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_68">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_65">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_87">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>等级名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>70</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red
</string>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addLevel</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addLevel</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
